Objects/codeobject.c

```diff
@@ -102,14 +102,27 @@ intern_string_constants(PyObject *tuple)
     return modified;
 }
 
-
 PyCodeObject *
-PyCode_New(int argcount, int posonlyargcount, int kwonlyargcount,
+PyCode_New(int argcount, int kwonlyargcount,
```

Comment:
nitpick: *here* i suggest to declare PyCode_New *after* PyCode_NewWithPosOnlyArgs declaration.